Update/omake_02.txt

```diff
@@ -1726,7 +1726,7 @@ void main()
 	PlaySE(4, "S20/09/440900118", 128, 64);
```

Comment:
The PS3 script is clearly different. I'll merge your changes and translate this line.